### DISCUSSION TITLE
Produce slate-compatible ast tree from markdown in nodejs (aka: md deserialization)

### DISCUSSION THREAD

**Discussion by giacomocerquone** [ianstormtaylor/slate#4117] [Q&A]
Hi all, I have a particular use case which doesn't seem to fit in your docs.
I'm working with graphCMS which, on creating programmatically a post, asks for a "rich text AST" produced by slate.

Now, I'm creating a node js migrator which reads content from md files and send them to the graphcms api, but right now I'm stuck at trying to create an AST, like the one slate creates, from the content of those md files.

Is there some method of this lib or some way to do this?
Thank you all.

**Comment by beorn** [comment]
You can check out https://github.com/hanford/remark-slate
